### DISCUSSION TITLE
amoled theme (pure black) when auto bright-dark is used

### DISCUSSION THREAD

**Discussion by ghost** [FossifyOrg/General-Discussion#148] [General]
If you customize Fossify colours you can make a pure black background for amoled display.

The same is not true with auto bright - dark, where you get a dark graish look.

Ideas to have in in the UI could be:

- add "auto bright / pure black" to the list
- or add a toggle "pure black mode" alongside the colour page

**Comment by Aga-C** [comment]
It's already requested here: https://github.com/FossifyOrg/General-Discussion/issues/109.
